code/__DEFINES/dcs/signals/signals_mod.dm

```diff
@@ -9,3 +9,16 @@
 #define COMSIG_MOD_MODULE_REMOVAL "mod_module_removal"
 	/// Cancels the removal of modules
 	#define MOD_CANCEL_REMOVAL (1 << 0)
+<<<<<<< HEAD
+=======
+/// Called when a module attempts to activate, however it does. At the end of checks so you can add some yourself, or work on trigger behavior (mob/user)
+#define COMSIG_MODULE_TRIGGERED "mod_module_triggered"
+	// Cancels activation, with no message. include feedback on your cancel.
+	#define MOD_ABORT_USE (1<<0)
+/// Called when a module activates, after all checks have passed and cooldown started.
+#define COMSIG_MODULE_ACTIVATED "mod_module_activated"
+/// Called when a module deactivates, after all checks have passed.
+#define COMSIG_MODULE_DEACTIVATED "mod_module_deactivated"
+/// Called when a module is used, after all checks have passed and cooldown started.
+#define COMSIG_MODULE_USED "mod_module_used"
+>>>>>>> fd9a7f8a581 (MOD update: Modular Cores (#64042))
```

Comment:
I honestly dont see the "Conflict" here, it just... didnt add these to the file right?

```suggestion

/// Called when a module attempts to activate, however it does. At the end of checks so you can add some yourself, or work on trigger behavior (mob/user)
#define COMSIG_MODULE_TRIGGERED "mod_module_triggered"
	// Cancels activation, with no message. include feedback on your cancel.
	#define MOD_ABORT_USE (1<<0)
/// Called when a module activates, after all checks have passed and cooldown started.
#define COMSIG_MODULE_ACTIVATED "mod_module_activated"
/// Called when a module deactivates, after all checks have passed.
#define COMSIG_MODULE_DEACTIVATED "mod_module_deactivated"
/// Called when a module is used, after all checks have passed and cooldown started.
#define COMSIG_MODULE_USED "mod_module_used"

```